resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
```

Comment:
@cprice404: @underscorgan and I were just talking about this. I'm not sure we fully understand the second option (the ezbake option).